### DISCUSSION TITLE
[Feature request] Custom repeat

### DISCUSSION THREAD

**Discussion by christaikobo** [donetick/donetick#22] [Ideas]
Dear Dev,

Is it possible to repeat a task every 2nd Sunday each month?

Thanks!

**Comment by meauxt** [comment]
Yes, I'm planning to improve the scheduler, but not immediately. It requires some careful consideration from the UI and UX

**Reply by hypervtechnics** [reply]
Would you consider also adding a repeat from completion date feature? Usually all apps are based on the due date. The tasks app for example has this feature and I find it quite useful. If I procrastinated something I should do every 10 days for 9 days and I finally did it and can tick it off, it should appear again in 10 days instead of tomorrow 😀

**Reply by meauxt** [reply]
donetick always has that, unless i misunderstood the ask 
<img width="857" alt="image" src="https://github.com/user-attachments/assets/abad2791-8ae1-4b53-9f8b-17494b3f3ad4">

**Reply by hypervtechnics** [reply]
Oh god, how could I not see this. My brain straight up skipped that part. Sorry. Thanks for the clarification tho.

**Comment by Danie10** [comment]
I have the identical requirement as the OP - I have a task that is scheduled for the first Wednesday of every month.
